### ISSUE TITLE
separate lock screen and wallpaper

### ISSUE THREAD

**Issue by AimVaguely** [RikudouSage/AiWallpaperChanger#109]
I would like the ability to give a separate prompt for my lock screen and my wallpaper. I like my lock screen and my wallpaper to have different themes.

**Comment by RikudouSage** [comment]
That needs a significant rewrite, so it's gonna take a while. But in the meantime, if you'd like to set your lock screen background in a different way (manually, using a different app...), you can enable live wallpaper in the app's settings and set it only for your homescreen.

**Comment by AimVaguely** [comment]
Yeah. I figured that was going to be the case. It is a feature request after all. I'll create a few more issues for other features that I (and hopefully other people) would like, but they are feature "requests" not feature "demands" so feel free to say if they don't match your vision of the app.
